### PR TITLE
Fixes, updated and improved the horrible German translation

### PR DESCRIPTION
**Pull request by Matti-Koopa**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -19,4 +19,10 @@ build
 
 # other
 eclipse
-run
```
Comment (AtomicBlom) [inline .gitignore:22]: Could I ask you to please give this a little cleanup?

**Review by AtomicBlom — CHANGES_REQUESTED** [review]

**Review by AtomicBlom — APPROVED** [review]

**Comment by AtomicBlom** [comment]
@Matti-Koopa Does this also fix https://github.com/TeamTwilight/twilightforest/issues/61?

**Comment by Matti-Koopa** [comment]
Yes it does.

**Comment by Tamaized** [comment]
(You can close your own issues)

**Pull request merged**